.github/workflows/vercel-preview-finance.yaml

```
@@ -0,0 +1,73 @@
+        description: 'Force deployment regardless of changes'
+        required: false
+        default: 'false'
+        type: boolean
+
+jobs:
+  check-changes:
+    runs-on: ubuntu-latest
+    if: (github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.head_branch != 'production') || github.event_name == 'workflow_dispatch'
+    outputs:
+      finance_detected: ${{ steps.check-finance.outputs.finance_detected }}
+      project_id: ${{ steps.extract-project-id.outputs.project_id }}
```

actionlint (1.7.7)

property "extract-project-id" is not defined in object type {check-finance: {conclusion: string; outcome: string; outputs: {string => string}}; download-artifact: {conclusion: string; outcome: string; outputs: {download-path: string}}}

(expression)